src/exojax/spec/multimol.py

```diff
@@ -0,0 +1,261 @@
+                                    auto_trange=auto_trange,
+                                    dit_grid_resolution=dit_grid_resolution)
+                opa_k.append(opa_i)
+            multiopa.append(opa_k)
+
+        return multiopa
+
+    def molmass(self):
+        """return molecular mass list and H and He 
+
+        Returns:
+            molmass_list: molecular mass list for self.mols_unique
+            molmassH2: molecular mass for hydorogen 
+            molmassHe: molecular mass for helium
+        """
+        from exojax.spec import molinfo
+
+        molmass_list = []
+        for i in range(len(self.mols_unique)):
+            molmass_list.append(molinfo.molmass(self.mols_unique[i]))
+        molmassH2 = molinfo.molmass("H2")
+        molmassHe = molinfo.molmass("He", db_HIT=False)
+
+        return molmass_list, molmassH2, molmassHe
+
+    
+
+def database_path_hitran12(simple_molecule_name):
+    """HITRAN12 default data path
+
+    Args:
+        simple_molecule_name (str): simple molecule name "H2O" 
+
+    Returns:
+        str: HITRAN12 default data path, such as "H2O/01_hit12.par" for "H2O"
+    """
+    from radis.db.classes import get_molecule_identifier
+    ihitran = get_molecule_identifier(simple_molecule_name)
+    return simple_molecule_name + "/" + str(ihitran).zfill(2) + "_hit12.par"
+
+
+def database_path_hitemp(simple_molname):
+    """default HITEMP path based on https://hitran.org/hitemp/
+
+    Args:
+        simple_molecule_name (str): simple molecule name "H2O" 
+
+    Returns:
+        str: HITEMP default data path, such as "H2O/01_HITEMP2010" for "H2O"
+    """
+    _hitemp_dbpath = {
+        "H2O": "H2O/01_HITEMP2010",
+        "CO2": "CO2/02_HITEMP2010",
+        "N2O": "N2O/04_HITEMP2019/04_HITEMP2019.par.bz2",
+        "CO": "CO/05_HITEMP2019/05_HITEMP2019.par.bz2",
+        "CH4": "CH4/06_HITEMP2020/06_HITEMP2020.par.bz2",
+        "NO": "NO/08_HITEMP2019/08_HITEMP2019.par.bz2",
+        "NO2": "NO2/10_HITEMP2019/10_HITEMP2019.par.bz2",
+        "OH": "OH/13_HITEMP2020/13_HITEMP2020.par.bz2"
+    }
+    return _hitemp_dbpath[simple_molname]
+
+
+def database_path_exomol(simple_molecule_name, database_root_path=".database"):
+    """default ExoMol path  
+
+    Args:
+        simple_molecule_name (str): simple molecule name "H2O" 
+        database_root_path (str, optional): root path of the database. Defaults to ".database".
+
+    Returns:
+        str: Exomol default data path
+    """
+    from exojax.utils.molname import simple_molname_to_exact_exomol_stable
+    from radis.api.exomolapi import get_exomol_database_list
+    exact_molname_exomol_stable = simple_molname_to_exact_exomol_stable(
+        simple_molecule_name)
+    mlist, recommended = get_exomol_database_list(simple_molecule_name,
+                                                  exact_molname_exomol_stable)
+    dbpath = database_root_path + "/" + simple_molecule_name + "/" + recommended
```

Comment:
This line caused a problem in my analysis. I think this should be the following.:
`dbpath = simple_molecule_name + "/" +  exact_molname_exomol_stable + "/" + recommended`

(For example, it reterns `./database/H2O/POKAZATEL` now, but it failed to download database. `H2O/1H2-16O/POKAZATEL` is correct.)

And we do not need `database_root_path` option for this function, like `database_path_hitemp` and `hitran`.

Reply:
Thanks! After correcting this line, did it work?

Reply:
Yes, it worked.

Reply:
corrected. Can you check if it works?

Reply:
Thank you! I confirmed that it works correctly.